alphagov-paas-grafana-annotation-resource.yml

```diff
@@ -0,0 +1,5 @@
+name: grafana-annotation-resource
```

Comment:
Hey @tlwr,
just a small change that I can do myself, we're intending to keep the name the same as you would use in a pipeline, so I will just remove the `-resource` from the name.

Let me know if this doesn't work for you and I will re-add it.